#! /bin/sh /usr/share/dpatch/dpatch-run
## 80-update-makefile.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: MakeProjectCreator generated Makefiles without examples and tests

@DPATCH@

diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/ACEXML/GNUmakefile ACE_wrappers/ACEXML/GNUmakefile
--- ACE_wrappers.orig/ACEXML/GNUmakefile	2005-08-05 15:07:20.000000000 +0200
+++ ACE_wrappers/ACEXML/GNUmakefile	2006-01-23 03:00:32.000000000 +0100
@@ -23,7 +23,5 @@
 $(REMAINING_TARGETS):
 	$(KEEP_GOING)@cd common && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd parser && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd examples && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd apps && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/apps/gperf/tests/GNUmakefile ACE_wrappers/apps/gperf/tests/GNUmakefile
--- ACE_wrappers.orig/apps/gperf/tests/GNUmakefile	2005-08-05 15:07:54.000000000 +0200
+++ ACE_wrappers/apps/gperf/tests/GNUmakefile	2006-01-23 03:00:35.000000000 +0100
@@ -19,57 +19,57 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: cppinset adainset cinset iinset iinset2 m3inset pinset preinset taoinset tinset
+all: pinset adainset cinset cppinset iinset iinset2 m3inset preinset taoinset tinset
 
-.PHONY: cppinset
-cppinset:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.cppinset
+.PHONY: pinset
+pinset:
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.pinset
 
 .PHONY: adainset
-adainset: cppinset
+adainset: pinset
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.adainset
 
 .PHONY: cinset
-cinset: cppinset
+cinset: pinset
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.cinset
 
+.PHONY: cppinset
+cppinset: pinset
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.cppinset
+
 .PHONY: iinset
-iinset: cppinset
+iinset: pinset
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.iinset
 
 .PHONY: iinset2
-iinset2: cppinset iinset
+iinset2: pinset iinset
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.iinset2
 
 .PHONY: m3inset
-m3inset: cppinset
+m3inset: pinset
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.m3inset
 
-.PHONY: pinset
-pinset: cppinset
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.pinset
-
 .PHONY: preinset
-preinset: cppinset
+preinset: pinset
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.preinset
 
 .PHONY: taoinset
-taoinset: cppinset
+taoinset: pinset
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.taoinset
 
 .PHONY: tinset
-tinset: cppinset
+tinset: pinset
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.tinset
 
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.cppinset $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.pinset $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.adainset $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.cinset $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.cppinset $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.iinset $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.iinset2 $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.m3inset $(@)
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.pinset $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.preinset $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.taoinset $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.tinset $(@)
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/ASNMP/GNUmakefile ACE_wrappers/ASNMP/GNUmakefile
--- ACE_wrappers.orig/ASNMP/GNUmakefile	2005-08-05 15:07:20.000000000 +0200
+++ ACE_wrappers/ASNMP/GNUmakefile	2006-01-23 03:00:31.000000000 +0100
@@ -23,6 +23,4 @@
 $(REMAINING_TARGETS):
 	$(KEEP_GOING)@cd asnmp && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd agent && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd examples && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/GNUmakefile ACE_wrappers/GNUmakefile
--- ACE_wrappers.orig/GNUmakefile	2005-08-05 15:07:54.000000000 +0200
+++ ACE_wrappers/GNUmakefile	2006-01-23 03:00:32.000000000 +0100
@@ -27,9 +27,6 @@
 	$(KEEP_GOING)@cd ACEXML && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd apps && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd netsvcs && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd performance-tests && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd protocols && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd examples && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd websvcs && $(MAKE) -f $(MAKEFILE) $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/Kokyu/GNUmakefile ACE_wrappers/Kokyu/GNUmakefile
--- ACE_wrappers.orig/Kokyu/GNUmakefile	2005-08-05 15:07:54.000000000 +0200
+++ ACE_wrappers/Kokyu/GNUmakefile	2006-01-23 03:00:36.000000000 +0100
@@ -20,7 +20,6 @@
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
 all: Kokyu
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 
 .PHONY: Kokyu
 Kokyu:
@@ -29,5 +28,4 @@
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Kokyu $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/protocols/GNUmakefile ACE_wrappers/protocols/GNUmakefile
--- ACE_wrappers.orig/protocols/GNUmakefile	2005-08-05 15:08:27.000000000 +0200
+++ ACE_wrappers/protocols/GNUmakefile	2006-01-23 03:00:36.000000000 +0100
@@ -22,6 +22,4 @@
 REMAINING_TARGETS := $(TARGETS_NESTED:.nested=) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
 	$(KEEP_GOING)@cd ace && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd examples && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/CIAO/DAnCE/GNUmakefile ACE_wrappers/TAO/CIAO/DAnCE/GNUmakefile
--- ACE_wrappers.orig/TAO/CIAO/DAnCE/GNUmakefile	2005-08-05 15:23:56.000000000 +0200
+++ ACE_wrappers/TAO/CIAO/DAnCE/GNUmakefile	2006-01-23 03:09:11.000000000 +0100
@@ -32,7 +32,5 @@
 	$(KEEP_GOING)@cd NodeApplication && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd Config_Handlers && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd Plan_Launcher && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd examples && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd new_RepositoryManager && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/GNUmakefile ACE_wrappers/TAO/GNUmakefile
--- ACE_wrappers.orig/TAO/GNUmakefile	2005-08-05 15:21:40.000000000 +0200
+++ ACE_wrappers/TAO/GNUmakefile	2006-01-23 03:07:16.000000000 +0100
@@ -24,10 +24,6 @@
 	$(KEEP_GOING)@cd TAO_IDL && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd tao && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd orbsvcs && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd examples && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd docs && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd performance-tests && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd interop-tests && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd utils && $(MAKE) -f $(MAKEFILE) $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/examples/CosEC/Factory/GNUmakefile ACE_wrappers/TAO/orbsvcs/examples/CosEC/Factory/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/examples/CosEC/Factory/GNUmakefile	2005-08-05 15:21:45.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/CosEC/Factory/GNUmakefile	2006-01-23 03:03:29.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: CosEC_Factory_Client CosEC_Factory_Server
-
-.PHONY: CosEC_Factory_Client
-CosEC_Factory_Client:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.CosEC_Factory_Client
+all: CosEC_Factory_Server CosEC_Factory_Client
 
 .PHONY: CosEC_Factory_Server
-CosEC_Factory_Server: CosEC_Factory_Client
+CosEC_Factory_Server:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.CosEC_Factory_Server
 
+.PHONY: CosEC_Factory_Client
+CosEC_Factory_Client: CosEC_Factory_Server
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.CosEC_Factory_Client
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.CosEC_Factory_Client $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.CosEC_Factory_Server $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.CosEC_Factory_Client $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/examples/ImR/Advanced/GNUmakefile ACE_wrappers/TAO/orbsvcs/examples/ImR/Advanced/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/examples/ImR/Advanced/GNUmakefile	2005-08-05 15:21:45.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/ImR/Advanced/GNUmakefile	2006-01-23 03:03:29.000000000 +0100
@@ -19,23 +19,23 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: Advanced_Client Advanced_Server Advanced_Manager
-
-.PHONY: Advanced_Client
-Advanced_Client:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Advanced_Client
+all: Advanced_Server Advanced_Client Advanced_Manager
 
 .PHONY: Advanced_Server
-Advanced_Server: Advanced_Client
+Advanced_Server:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Advanced_Server
 
+.PHONY: Advanced_Client
+Advanced_Client: Advanced_Server
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Advanced_Client
+
 .PHONY: Advanced_Manager
-Advanced_Manager: Advanced_Client Advanced_Server
+Advanced_Manager: Advanced_Server
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Advanced_Manager
 
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Advanced_Client $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Advanced_Server $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Advanced_Client $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Advanced_Manager $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/examples/Security/Send_File/GNUmakefile ACE_wrappers/TAO/orbsvcs/examples/Security/Send_File/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/examples/Security/Send_File/GNUmakefile	2005-08-05 15:21:45.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/Security/Send_File/GNUmakefile	2006-01-23 03:03:30.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: Security_Send_File_Server Security_Send_File_Client
-
-.PHONY: Security_Send_File_Server
-Security_Send_File_Server:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Security_Send_File_Server
+all: Security_Send_File_Client Security_Send_File_Server
 
 .PHONY: Security_Send_File_Client
-Security_Send_File_Client: Security_Send_File_Server
+Security_Send_File_Client:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Security_Send_File_Client
 
+.PHONY: Security_Send_File_Server
+Security_Send_File_Server: Security_Send_File_Client
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Security_Send_File_Server
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Security_Send_File_Server $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Security_Send_File_Client $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Security_Send_File_Server $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/Naming_Service/GNUmakefile ACE_wrappers/TAO/orbsvcs/Naming_Service/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/Naming_Service/GNUmakefile	2005-08-05 15:14:21.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/Naming_Service/GNUmakefile	2006-01-23 03:03:29.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: Naming_Service NT_Naming_Service
-
-.PHONY: Naming_Service
-Naming_Service:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Naming_Service
+all: NT_Naming_Service Naming_Service
 
 .PHONY: NT_Naming_Service
-NT_Naming_Service: Naming_Service
+NT_Naming_Service:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.NT_Naming_Service
 
+.PHONY: Naming_Service
+Naming_Service: NT_Naming_Service
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Naming_Service
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Naming_Service $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.NT_Naming_Service $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Naming_Service $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/Notify_Service/GNUmakefile ACE_wrappers/TAO/orbsvcs/Notify_Service/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/Notify_Service/GNUmakefile	2005-08-05 15:14:21.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/Notify_Service/GNUmakefile	2006-01-23 03:03:29.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: NT_Notify_Service Notify_Service
-
-.PHONY: NT_Notify_Service
-NT_Notify_Service:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.NT_Notify_Service
+all: Notify_Service NT_Notify_Service
 
 .PHONY: Notify_Service
-Notify_Service: NT_Notify_Service
+Notify_Service:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Notify_Service
 
+.PHONY: NT_Notify_Service
+NT_Notify_Service: Notify_Service
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.NT_Notify_Service
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.NT_Notify_Service $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Notify_Service $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.NT_Notify_Service $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/GNUmakefile.CosNotification_Serv ACE_wrappers/TAO/orbsvcs/orbsvcs/GNUmakefile.CosNotification_Serv
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/GNUmakefile.CosNotification_Serv	2005-08-05 15:12:55.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/GNUmakefile.CosNotification_Serv	2006-01-23 03:02:47.000000000 +0100
@@ -78,6 +78,7 @@
   Notify/Routing_Slip.cpp \
   Notify/Routing_Slip_Persistence_Manager.cpp \
   Notify/Routing_Slip_Queue.cpp \
+  Notify/Service.cpp \
   Notify/Subscription_Change_Worker.cpp \
   Notify/Supplier.cpp \
   Notify/SupplierAdmin.cpp \
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/GNUmakefile.PortableGroup ACE_wrappers/TAO/orbsvcs/orbsvcs/GNUmakefile.PortableGroup
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/GNUmakefile.PortableGroup	2005-08-05 15:12:41.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/GNUmakefile.PortableGroup	2006-01-23 03:02:29.000000000 +0100
@@ -26,9 +26,9 @@
 PortableGroup = \
   miopC.cpp \
   miopS.cpp \
+  PortableGroup_Simple_DSC.cpp \
   PortableGroupC.cpp \
   PortableGroupS.cpp \
-  PortableGroup_Simple_DSC.cpp \
   PortableGroup/MIOP.cpp \
   PortableGroup/GOA.cpp \
   PortableGroup/PG_Utils.cpp \
@@ -353,7 +353,7 @@
 
 # This assignment forces make to run the idl_stubs
 # target before building any of the source files.
-FORCED_IDL_STUBS = miopC.cpp miopS.cpp PortableGroupC.cpp PortableGroupS.cpp PortableGroup_Simple_DSC.cpp PortableGroup/MIOP.cpp PortableGroup/GOA.cpp PortableGroup/PG_Utils.cpp PortableGroup/PG_Default_Property_Validator.cpp PortableGroup/PG_FactoryRegistry.cpp PortableGroup/PG_Factory_Map.cpp PortableGroup/PG_Factory_Set.cpp PortableGroup/PG_GenericFactory.cpp PortableGroup/PG_Group_Factory.cpp PortableGroup/PG_Group_Guard.cpp PortableGroup/PG_Location_Equal_To.cpp PortableGroup/PG_Location_Hash.cpp PortableGroup/PG_Location_Map.cpp PortableGroup/PG_MemberInfo.cpp PortableGroup/PG_Null_Property_Validator.cpp PortableGroup/PG_ObjectGroupManager.cpp PortableGroup/PG_ObjectGroup_Map.cpp PortableGroup/PG_Object_Adapter_Factory.cpp PortableGroup/PG_Object_Group.cpp PortableGroup/PG_Object_Group_Manipulator.cpp PortableGroup/PG_Operators.cpp PortableGroup/PG_Properties_Encoder.cpp PortableGroup/PG_Properties_Support.cpp PortableGroup/PG_PropertyManager.cpp PortableGroup/PG_Property_Set.cpp PortableGroup/PG_Property_Utils.cpp PortableGroup/PG_Servant_Dispatcher.cpp PortableGroup/PortableGroup_Acceptor_Registry.cpp PortableGroup/PortableGroup_Loader.cpp PortableGroup/PortableGroup_ORBInitializer.cpp PortableGroup/PortableGroup_Request_Dispatcher.cpp PortableGroup/Portable_Group_Map.cpp PortableGroup/UIPMC_Acceptor.cpp PortableGroup/UIPMC_Connection_Handler.cpp PortableGroup/UIPMC_Connector.cpp PortableGroup/UIPMC_Endpoint.cpp PortableGroup/UIPMC_Factory.cpp PortableGroup/UIPMC_Message_Block_Data_Iterator.cpp PortableGroup/UIPMC_Profile.cpp PortableGroup/UIPMC_Transport.cpp PortableGroup/UIPMC_Wait_Never.cpp
+FORCED_IDL_STUBS = miopC.cpp miopS.cpp PortableGroup_Simple_DSC.cpp PortableGroupC.cpp PortableGroupS.cpp PortableGroup/MIOP.cpp PortableGroup/GOA.cpp PortableGroup/PG_Utils.cpp PortableGroup/PG_Default_Property_Validator.cpp PortableGroup/PG_FactoryRegistry.cpp PortableGroup/PG_Factory_Map.cpp PortableGroup/PG_Factory_Set.cpp PortableGroup/PG_GenericFactory.cpp PortableGroup/PG_Group_Factory.cpp PortableGroup/PG_Group_Guard.cpp PortableGroup/PG_Location_Equal_To.cpp PortableGroup/PG_Location_Hash.cpp PortableGroup/PG_Location_Map.cpp PortableGroup/PG_MemberInfo.cpp PortableGroup/PG_Null_Property_Validator.cpp PortableGroup/PG_ObjectGroupManager.cpp PortableGroup/PG_ObjectGroup_Map.cpp PortableGroup/PG_Object_Adapter_Factory.cpp PortableGroup/PG_Object_Group.cpp PortableGroup/PG_Object_Group_Manipulator.cpp PortableGroup/PG_Operators.cpp PortableGroup/PG_Properties_Encoder.cpp PortableGroup/PG_Properties_Support.cpp PortableGroup/PG_PropertyManager.cpp PortableGroup/PG_Property_Set.cpp PortableGroup/PG_Property_Utils.cpp PortableGroup/PG_Servant_Dispatcher.cpp PortableGroup/PortableGroup_Acceptor_Registry.cpp PortableGroup/PortableGroup_Loader.cpp PortableGroup/PortableGroup_ORBInitializer.cpp PortableGroup/PortableGroup_Request_Dispatcher.cpp PortableGroup/Portable_Group_Map.cpp PortableGroup/UIPMC_Acceptor.cpp PortableGroup/UIPMC_Connection_Handler.cpp PortableGroup/UIPMC_Connector.cpp PortableGroup/UIPMC_Endpoint.cpp PortableGroup/UIPMC_Factory.cpp PortableGroup/UIPMC_Message_Block_Data_Iterator.cpp PortableGroup/UIPMC_Profile.cpp PortableGroup/UIPMC_Transport.cpp PortableGroup/UIPMC_Wait_Never.cpp
 FORCED_IDL_STUBS := $(FORCED_IDL_STUBS:miopC.cpp=)
 FORCED_IDL_STUBS := $(FORCED_IDL_STUBS:miopS.cpp=)
 FORCED_IDL_STUBS := $(FORCED_IDL_STUBS:PortableGroup_Simple_DSC.cpp=)
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/performance-tests/RTEvent/RTCORBA_Baseline/GNUmakefile ACE_wrappers/TAO/orbsvcs/performance-tests/RTEvent/RTCORBA_Baseline/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/performance-tests/RTEvent/RTCORBA_Baseline/GNUmakefile	2005-08-05 15:21:48.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/performance-tests/RTEvent/RTCORBA_Baseline/GNUmakefile	2006-01-23 03:03:33.000000000 +0100
@@ -19,23 +19,23 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: RTCORBA_Baseline_IDL RTCORBA_Baseline_Server RTCORBA_Baseline_Client
+all: RTCORBA_Baseline_IDL RTCORBA_Baseline_Client RTCORBA_Baseline_Server
 
 .PHONY: RTCORBA_Baseline_IDL
 RTCORBA_Baseline_IDL:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.RTCORBA_Baseline_IDL
 
-.PHONY: RTCORBA_Baseline_Server
-RTCORBA_Baseline_Server: RTCORBA_Baseline_IDL
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.RTCORBA_Baseline_Server
-
 .PHONY: RTCORBA_Baseline_Client
-RTCORBA_Baseline_Client: RTCORBA_Baseline_IDL RTCORBA_Baseline_Server
+RTCORBA_Baseline_Client: RTCORBA_Baseline_IDL
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.RTCORBA_Baseline_Client
 
+.PHONY: RTCORBA_Baseline_Server
+RTCORBA_Baseline_Server: RTCORBA_Baseline_IDL RTCORBA_Baseline_Client
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.RTCORBA_Baseline_Server
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.RTCORBA_Baseline_IDL $(@)
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.RTCORBA_Baseline_Server $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.RTCORBA_Baseline_Client $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.RTCORBA_Baseline_Server $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/EC_Throughput/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/EC_Throughput/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/EC_Throughput/GNUmakefile	2005-08-05 15:21:49.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/EC_Throughput/GNUmakefile	2006-01-23 03:03:34.000000000 +0100
@@ -19,23 +19,23 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: ECT_Consumer ECT_Throughput ECT_Supplier
+all: ECT_Supplier ECT_Consumer ECT_Throughput
+
+.PHONY: ECT_Supplier
+ECT_Supplier:
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.ECT_Supplier
 
 .PHONY: ECT_Consumer
-ECT_Consumer:
+ECT_Consumer: ECT_Supplier
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.ECT_Consumer
 
 .PHONY: ECT_Throughput
-ECT_Throughput: ECT_Consumer
+ECT_Throughput: ECT_Supplier ECT_Consumer
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.ECT_Throughput
 
-.PHONY: ECT_Supplier
-ECT_Supplier: ECT_Consumer ECT_Throughput
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.ECT_Supplier
-
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.ECT_Supplier $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.ECT_Consumer $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.ECT_Throughput $(@)
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.ECT_Supplier $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/FaultTolerance/GroupRef_Manipulation/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/FaultTolerance/GroupRef_Manipulation/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/FaultTolerance/GroupRef_Manipulation/GNUmakefile	2005-08-05 15:21:50.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/FaultTolerance/GroupRef_Manipulation/GNUmakefile	2006-01-23 03:03:35.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: GroupRef_Manipulation_Client GroupRef_Manipulation_Server
-
-.PHONY: GroupRef_Manipulation_Client
-GroupRef_Manipulation_Client:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.GroupRef_Manipulation_Client
+all: GroupRef_Manipulation_Server GroupRef_Manipulation_Client
 
 .PHONY: GroupRef_Manipulation_Server
-GroupRef_Manipulation_Server: GroupRef_Manipulation_Client
+GroupRef_Manipulation_Server:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.GroupRef_Manipulation_Server
 
+.PHONY: GroupRef_Manipulation_Client
+GroupRef_Manipulation_Client: GroupRef_Manipulation_Server
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.GroupRef_Manipulation_Client
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.GroupRef_Manipulation_Client $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.GroupRef_Manipulation_Server $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.GroupRef_Manipulation_Client $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/HTIOP/BiDirectional/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/HTIOP/BiDirectional/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/HTIOP/BiDirectional/GNUmakefile	2005-08-05 15:21:50.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/HTIOP/BiDirectional/GNUmakefile	2006-01-23 03:03:35.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: HTIOP_BiDirectional_Server HTIOP_BiDirectional_Client
-
-.PHONY: HTIOP_BiDirectional_Server
-HTIOP_BiDirectional_Server:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_BiDirectional_Server
+all: HTIOP_BiDirectional_Client HTIOP_BiDirectional_Server
 
 .PHONY: HTIOP_BiDirectional_Client
-HTIOP_BiDirectional_Client: HTIOP_BiDirectional_Server
+HTIOP_BiDirectional_Client:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_BiDirectional_Client
 
+.PHONY: HTIOP_BiDirectional_Server
+HTIOP_BiDirectional_Server: HTIOP_BiDirectional_Client
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_BiDirectional_Server
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_BiDirectional_Server $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_BiDirectional_Client $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_BiDirectional_Server $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/HTIOP/Hello/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/HTIOP/Hello/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/HTIOP/Hello/GNUmakefile	2005-08-05 15:21:50.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/HTIOP/Hello/GNUmakefile	2006-01-23 03:03:35.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: HTIOP_Hello_Client HTIOP_Hello_Server
-
-.PHONY: HTIOP_Hello_Client
-HTIOP_Hello_Client:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_Hello_Client
+all: HTIOP_Hello_Server HTIOP_Hello_Client
 
 .PHONY: HTIOP_Hello_Server
-HTIOP_Hello_Server: HTIOP_Hello_Client
+HTIOP_Hello_Server:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_Hello_Server
 
+.PHONY: HTIOP_Hello_Client
+HTIOP_Hello_Client: HTIOP_Hello_Server
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_Hello_Client
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_Hello_Client $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_Hello_Server $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.HTIOP_Hello_Client $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/ImplRepo/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/ImplRepo/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/ImplRepo/GNUmakefile	2005-08-05 15:21:51.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/ImplRepo/GNUmakefile	2006-01-23 03:03:35.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: airplane_server airplane_client nestea_server nestea_client
+all: airplane_client airplane_server nestea_server nestea_client
 	$(KEEP_GOING)@cd NameService && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd scale && $(MAKE) -f $(MAKEFILE) $(@)
 
-.PHONY: airplane_server
-airplane_server:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.airplane_server
-
 .PHONY: airplane_client
-airplane_client: airplane_server
+airplane_client:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.airplane_client
 
+.PHONY: airplane_server
+airplane_server: airplane_client
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.airplane_server
+
 .PHONY: nestea_server
 nestea_server:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.nestea_server
@@ -41,8 +41,8 @@
 
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.airplane_server $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.airplane_client $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.airplane_server $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.nestea_server $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.nestea_client $(@)
 	$(KEEP_GOING)@cd NameService && $(MAKE) -f $(MAKEFILE) $(@)
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/ImplRepo/scale/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/ImplRepo/scale/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/ImplRepo/scale/GNUmakefile	2005-08-05 15:21:51.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/ImplRepo/scale/GNUmakefile	2006-01-23 03:03:35.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: scaletest_client scaletest_server
-
-.PHONY: scaletest_client
-scaletest_client:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.scaletest_client
+all: scaletest_server scaletest_client
 
 .PHONY: scaletest_server
-scaletest_server: scaletest_client
+scaletest_server:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.scaletest_server
 
+.PHONY: scaletest_client
+scaletest_client: scaletest_server
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.scaletest_client
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.scaletest_client $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.scaletest_server $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.scaletest_client $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/Notify/Ordering/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/Notify/Ordering/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/Notify/Ordering/GNUmakefile	2005-08-05 15:21:51.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/Notify/Ordering/GNUmakefile	2006-01-23 03:03:35.000000000 +0100
@@ -19,28 +19,28 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: Ordering_Ntf_Seq_Cons Ordering_Ntf_Seq_Supp Ordering_Ntf_Struct_Cons Ordering_Ntf_Struct_Supp
-
-.PHONY: Ordering_Ntf_Seq_Cons
-Ordering_Ntf_Seq_Cons:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Seq_Cons
+all: Ordering_Ntf_Seq_Supp Ordering_Ntf_Seq_Cons Ordering_Ntf_Struct_Cons Ordering_Ntf_Struct_Supp
 
 .PHONY: Ordering_Ntf_Seq_Supp
-Ordering_Ntf_Seq_Supp: Ordering_Ntf_Seq_Cons
+Ordering_Ntf_Seq_Supp:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Seq_Supp
 
+.PHONY: Ordering_Ntf_Seq_Cons
+Ordering_Ntf_Seq_Cons: Ordering_Ntf_Seq_Supp
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Seq_Cons
+
 .PHONY: Ordering_Ntf_Struct_Cons
-Ordering_Ntf_Struct_Cons: Ordering_Ntf_Seq_Cons
+Ordering_Ntf_Struct_Cons: Ordering_Ntf_Seq_Supp Ordering_Ntf_Seq_Cons
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Struct_Cons
 
 .PHONY: Ordering_Ntf_Struct_Supp
-Ordering_Ntf_Struct_Supp: Ordering_Ntf_Seq_Cons Ordering_Ntf_Seq_Supp
+Ordering_Ntf_Struct_Supp: Ordering_Ntf_Seq_Supp
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Struct_Supp
 
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Seq_Cons $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Seq_Supp $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Seq_Cons $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Struct_Cons $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Ordering_Ntf_Struct_Supp $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/Notify/Sequence_Multi_ETCL_Filter/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/Notify/Sequence_Multi_ETCL_Filter/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/Notify/Sequence_Multi_ETCL_Filter/GNUmakefile	2005-08-05 15:21:51.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/Notify/Sequence_Multi_ETCL_Filter/GNUmakefile	2006-01-23 03:03:35.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: Seq_Multi_ETCL_Filter_Ntf_Seq_Cons Seq_Multi_ETCL_Filter_Ntf_Seq_Supp
-
-.PHONY: Seq_Multi_ETCL_Filter_Ntf_Seq_Cons
-Seq_Multi_ETCL_Filter_Ntf_Seq_Cons:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_ETCL_Filter_Ntf_Seq_Cons
+all: Seq_Multi_ETCL_Filter_Ntf_Seq_Supp Seq_Multi_ETCL_Filter_Ntf_Seq_Cons
 
 .PHONY: Seq_Multi_ETCL_Filter_Ntf_Seq_Supp
-Seq_Multi_ETCL_Filter_Ntf_Seq_Supp: Seq_Multi_ETCL_Filter_Ntf_Seq_Cons
+Seq_Multi_ETCL_Filter_Ntf_Seq_Supp:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_ETCL_Filter_Ntf_Seq_Supp
 
+.PHONY: Seq_Multi_ETCL_Filter_Ntf_Seq_Cons
+Seq_Multi_ETCL_Filter_Ntf_Seq_Cons: Seq_Multi_ETCL_Filter_Ntf_Seq_Supp
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_ETCL_Filter_Ntf_Seq_Cons
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_ETCL_Filter_Ntf_Seq_Cons $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_ETCL_Filter_Ntf_Seq_Supp $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_ETCL_Filter_Ntf_Seq_Cons $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/Notify/Sequence_Multi_Filter/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/Notify/Sequence_Multi_Filter/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/Notify/Sequence_Multi_Filter/GNUmakefile	2005-08-05 15:21:51.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/Notify/Sequence_Multi_Filter/GNUmakefile	2006-01-23 03:03:35.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: Seq_Multi_Filter_Ntf_Seq_Cons Seq_Multi_Filter_Ntf_Seq_Supp
-
-.PHONY: Seq_Multi_Filter_Ntf_Seq_Cons
-Seq_Multi_Filter_Ntf_Seq_Cons:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_Filter_Ntf_Seq_Cons
+all: Seq_Multi_Filter_Ntf_Seq_Supp Seq_Multi_Filter_Ntf_Seq_Cons
 
 .PHONY: Seq_Multi_Filter_Ntf_Seq_Supp
-Seq_Multi_Filter_Ntf_Seq_Supp: Seq_Multi_Filter_Ntf_Seq_Cons
+Seq_Multi_Filter_Ntf_Seq_Supp:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_Filter_Ntf_Seq_Supp
 
+.PHONY: Seq_Multi_Filter_Ntf_Seq_Cons
+Seq_Multi_Filter_Ntf_Seq_Cons: Seq_Multi_Filter_Ntf_Seq_Supp
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_Filter_Ntf_Seq_Cons
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_Filter_Ntf_Seq_Cons $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_Filter_Ntf_Seq_Supp $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Seq_Multi_Filter_Ntf_Seq_Cons $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/Security/BiDirectional/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/Security/BiDirectional/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/Security/BiDirectional/GNUmakefile	2005-08-05 15:21:52.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/Security/BiDirectional/GNUmakefile	2006-01-23 03:03:37.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: Secure_BiDir_SSL_Client Secure_BiDir_SSL_Server
-
-.PHONY: Secure_BiDir_SSL_Client
-Secure_BiDir_SSL_Client:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Secure_BiDir_SSL_Client
+all: Secure_BiDir_SSL_Server Secure_BiDir_SSL_Client
 
 .PHONY: Secure_BiDir_SSL_Server
-Secure_BiDir_SSL_Server: Secure_BiDir_SSL_Client
+Secure_BiDir_SSL_Server:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Secure_BiDir_SSL_Server
 
+.PHONY: Secure_BiDir_SSL_Client
+Secure_BiDir_SSL_Client: Secure_BiDir_SSL_Server
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Secure_BiDir_SSL_Client
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Secure_BiDir_SSL_Client $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Secure_BiDir_SSL_Server $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Secure_BiDir_SSL_Client $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/Security/Big_Request/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/Security/Big_Request/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/Security/Big_Request/GNUmakefile	2005-08-05 15:21:52.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/Security/Big_Request/GNUmakefile	2006-01-23 03:03:37.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: Big_Request_Security_Server Big_Request_Security_Client
-
-.PHONY: Big_Request_Security_Server
-Big_Request_Security_Server:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Big_Request_Security_Server
+all: Big_Request_Security_Client Big_Request_Security_Server
 
 .PHONY: Big_Request_Security_Client
-Big_Request_Security_Client: Big_Request_Security_Server
+Big_Request_Security_Client:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Big_Request_Security_Client
 
+.PHONY: Big_Request_Security_Server
+Big_Request_Security_Server: Big_Request_Security_Client
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Big_Request_Security_Server
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Big_Request_Security_Server $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Big_Request_Security_Client $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.Big_Request_Security_Server $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/TAO/orbsvcs/tests/Security/MT_SSLIOP/GNUmakefile ACE_wrappers/TAO/orbsvcs/tests/Security/MT_SSLIOP/GNUmakefile
--- ACE_wrappers.orig/TAO/orbsvcs/tests/Security/MT_SSLIOP/GNUmakefile	2005-08-05 15:21:52.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/tests/Security/MT_SSLIOP/GNUmakefile	2006-01-23 03:03:37.000000000 +0100
@@ -19,18 +19,18 @@
 
 include $(ACE_ROOT)/include/makeinclude/macros.GNU
 
-all: MT_SSLIOP_Security_Client MT_SSLIOP_Security_Server
-
-.PHONY: MT_SSLIOP_Security_Client
-MT_SSLIOP_Security_Client:
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.MT_SSLIOP_Security_Client
+all: MT_SSLIOP_Security_Server MT_SSLIOP_Security_Client
 
 .PHONY: MT_SSLIOP_Security_Server
-MT_SSLIOP_Security_Server: MT_SSLIOP_Security_Client
+MT_SSLIOP_Security_Server:
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.MT_SSLIOP_Security_Server
 
+.PHONY: MT_SSLIOP_Security_Client
+MT_SSLIOP_Security_Client: MT_SSLIOP_Security_Server
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.MT_SSLIOP_Security_Client
+
 REMAINING_TARGETS := $(subst all, , $(TARGETS_NESTED:.nested=)) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
-	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.MT_SSLIOP_Security_Client $(@)
 	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.MT_SSLIOP_Security_Server $(@)
+	$(KEEP_GOING)@$(MAKE) -f GNUmakefile.MT_SSLIOP_Security_Client $(@)
 
diff -I '^# .*bin/mwc.pl.*-include.*-recurse.*' -Nru ACE_wrappers.orig/websvcs/GNUmakefile ACE_wrappers/websvcs/GNUmakefile
--- ACE_wrappers.orig/websvcs/GNUmakefile	2005-08-05 15:07:30.000000000 +0200
+++ ACE_wrappers/websvcs/GNUmakefile	2006-01-23 03:00:32.000000000 +0100
@@ -22,5 +22,4 @@
 REMAINING_TARGETS := $(TARGETS_NESTED:.nested=) $(CUSTOM_TARGETS)
 $(REMAINING_TARGETS):
 	$(KEEP_GOING)@cd lib && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
 
